tests/kokkos-based/helpers.hpp

```diff
@@ -47,6 +47,12 @@
 #include <experimental/mdspan>
 #include <random>
 
+#if KOKKOS_VERSION <= 30600
```

Comment:
This would fail with the upcoming patch release 3.6.1

Reply:
Thanks! Should the number be `30700` then or can that change before 3.7 ?

Reply:
Is there a smart way to solve it ?
For instance can one include whole `Experimental` into `Kokkos` namespace regardless of version ?
(we'd be doing that for tests only here)

Reply:
I think we are probably better off waiting for 3.6.1 and in the meantime we just use Kokkos develop

Reply:
I'd go for
```suggestion
#if KOKKOS_VERSION < 30699
```